---
strands_perception_msgs/Table table
